### PR TITLE
container: add quorum docker tools

### PR DESCRIPTION
**Pull request by yutelin**

### CODE REVIEW

File: tests/quorum/functional/general_consensus_test.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	})
+
+	AfterEach(func() {
+		blockchain.Stop(true)
+		blockchain.Finalize()
+		constellationNetwork.Stop()
+		constellationNetwork.Finalize()
+	})
+
+	FIt("do nothing", func() {
```
Comment (bailantaotao) [inline tests/quorum/functional/general_consensus_test.go]: remove?

File: tests/quorum/functional/general_consensus_test.go [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	// 	}
+
+	// 	for i := 0; i < len(blockchain.Validators()); i++ {
+	// 		err := <-errc
+	// 		Expect(err).To(BeNil())
+	// 	}
+
+	// })
+
+	FIt("QFS-01-03: Peer connection", func(done Done) {
```
Comment (bailantaotao) [inline tests/quorum/functional/general_consensus_test.go]: remove F

File: container/blockchain.go [hunk 3]
```diff
@@ -386,3 +441,129 @@ func (bc *blockchain) stop(validators []Ethereum, force bool) error {
 	}
 	return nil
 }
+
+// Constellation functions ----------------------------------------------------------------------------
+type ConstellationNetwork interface {
+	Start() error
+	Stop() error
+	Finalize()
+	NumOfValidators() int
```
Comment (bailantaotao) [inline container/blockchain.go]: NumOfConstellations()

File: container/blockchain.go [hunk 4]
```diff
@@ -386,3 +441,129 @@ func (bc *blockchain) stop(validators []Ethereum, force bool) error {
...
+}
+
+func (ctn *constellationNetwork) Finalize() {
+	// Clean up local working directory
+	for _, ct := range ctn.constellations {
+		os.RemoveAll(ct.WorkDir())
+	}
+}
+
+func (ctn *constellationNetwork) NumOfValidators() int {
```
Comment (bailantaotao) [inline container/blockchain.go]: NumOfConstellations()

**Comment by yutelin** [comment]
Fixed the comments.

**Pull request merged**